hydpy/models/rconc_convolution.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# pylint: disable=line-too-long, unused-wildcard-import
+"""Calculate runoff concentration by convolution of the unit hydrograph
+
+|rconc_convolution| is a submodel that supplies its main model with the
+calculation of the runoff concentration by convolution of the unit hydrograph
+"""
```

Comment:
If we do not demonstrate a submodel's functionality by integration tests in its own module, we should at least add links to the integration tests where a main model uses the submodel.

Also, "convolution" and "unit hydrograph" sound slightly general, while the given submodel deals with the specific HBV96 triangle unit hydrograph.  So, we need to add some information.  Maybe we should strive for consistency with the documentation of `evap_pet_hbv96` and `evap_aet_hbv96`.

Reply:
I agree, the shape of the unit hydrograph should be reflected in the name. But what is specific to HBV96, other than the triangle being isosceles?

Reply:
The triangle itself is not the only possible shape (and I suspect not even a very common one).  Other (I think, more common) choices, that we already implemented, are to configure the Unit Hydrograph based on the [linear storage cascade](https://hydpy-dev.github.io/hydpy/4.0/iuhtools.html#hydpy.auxs.iuhtools.LinearStorageCascade) or the [translation diffusion equation](https://hydpy-dev.github.io/hydpy/4.0/iuhtools.html#hydpy.auxs.iuhtools.TranslationDiffusionEquation).